Cargo.toml

```diff
@@ -19,16 +19,13 @@ moore-common = { path = "src/common", version = "0.8.0-dev" }
 moore-derive = { path = "src/derive", version = "0.8.0-dev" }
 moore-svlog = { path = "src/svlog", version = "0.8.0-dev" }
 moore-vhdl = { path = "src/vhdl", version = "0.8.0-dev" }
-bincode = "0.6.1"
 clap = "2"
 # llhd = { git = "https://github.com/fabianschuiki/llhd", rev = "master", version = "0.9" }
 llhd = "0.9"
 num = "0.1"
-rustc-serialize = "0.3.22"
+rustc-serialize = "0.3.24"
```

Comment:
I think it would be good to eventually get rid of `rustc-serialize`. Really no point of having it when `serde` is around, and currently none of the data structures it is used on get serialized to disk. I think I'll open an issue.